worker-message-prioritization-distribution-container/src/test/java/com/github/workerframework/workermessageprioritization/redistribution/LowLevelDistributorIT.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022-2023 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.workerframework.workermessageprioritization.redistribution;
+
+import com.github.workerframework.workermessageprioritization.rabbitmq.Queue;
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.concurrent.TimeoutException;
+
+public final class LowLevelDistributorIT extends DistributorTestBase {
```

Comment:
This test is almost the same as [LowLevelDistributorIT.java](https://github.com/WorkerFramework/worker-message-prioritization/blob/main/worker-message-prioritization-distribution/src/test/java/com/github/workerframework/workermessageprioritization/redistribution/LowLevelDistributorIT.java), except that it is not calling any domain objects, its just sending messages to rabbit. I don't know if we still want to keep both sets of tests.